Formula/w/wget2.rb

```diff
@@ -1,10 +1,15 @@
 class Wget2 < Formula
   desc "Successor of GNU Wget, a file and recursive website downloader"
   homepage "https://gitlab.com/gnuwget/wget2"
-  url "https://gitlab.com/gnuwget/wget2/uploads/83752270de83e103306576e67a1c7c80/wget2-2.0.1.tar.gz"
-  sha256 "0bb7fa03697bb5b8d05e1b5e15b863440826eb845874c4ffb5e32330f9845db1"
+  url "https://ftp.gnu.org/gnu/wget/wget2-2.1.0.tar.gz"
+  sha256 "a05dc5191c6bad9313fd6db2777a78f5527ba4774f665d5d69f5a7461b49e2e7"
   license "GPL-3.0-or-later"
 
+  livecheck do
+    url "https://ftp.gnu.org/gnu/wget/"
```

Comment:
```suggestion
    url :stable
```

Sorry for the additional review but I overlooked this previously. By default, livecheck will use the `Gnu` strategy with the `stable` URL and check the same URL that's used in this `livecheck` block, so we simply use `url :stable` here.

Reply:
I'm somewhat hesitant to do that because the stable URL scheme changed just from 2.1.0.
Before the version the stable URL points to GitLab upload, and I'm not sure if the scheme change is permanent or not. 
The duplicate URL is a future-proof for when the URL won't point to the GNU page in the later version.

Reply:
I understand your concern but if the `stable` URL changes to a different source in a subsequent PR, then the `livecheck` block would need to be updated in the same PR [to align with the new `stable` source] regardless. Using `url :stable` is more likely to cause livecheck to return an `Unable to get versions` error if the `stable` URL changes to a different source and this error will be surfaced from our CI. The error would need to be addressed before the PR is merged, so it helps to ensure the `livecheck` block is kept in line with the `stable` source.

Using a string literal that duplicates the URL from the `Gnu` strategy _would_ allow the check to continue to work if the `stable` URL changes to a different source but that's not actually desirable for us (as counterintuitive as it may seem). In that scenario, a PR changing the `stable` source may pass CI and end up being merged despite the `livecheck` block not checking the `stable` source. This type of issue is harder to identify than a check that's visibly failing.

Long story short, we should use `url :stable` here and if the `stable` source changes again in the future, we will simply update (or remove) the `livecheck` block at the same time. This is standard procedure for `livecheck` blocks in both homebrew/core and homebrew/cask and this one isn't an exception.

Does this make sense?

Reply:
It perfectly makes sense, thank you for the detailed explanation!

> The error would need to be addressed before the PR is merged

I didn't know this.
If actions can catch the error, I'm fine with the change.

Reply:
The change is applied and squashed.